frontend/src/features/boards/components/BoardCard.vue

```diff
@@ -0,0 +1,65 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import { useRouter } from "vue-router";
+
+import { state } from "@/modules/store";
+import { Board } from "@f/boards/types";
```

Comment:
```suggestion
import type { Board } from "@f/boards/types";
```

Type imports should be prefixed by the `type` keyword. This ensures they are not imported at runtime.